target/linux/ramips/image/mt7621.mk

```diff
@@ -262,6 +262,14 @@ define Device/totolink_a7000r
 endef
 TARGET_DEVICES += totolink_a7000r
 
+define Device/adslr-g7
```

Comment:
Please name the device with vendor_model scheme, like for the DTS compatible.

However, maybe clarify what's model name and what's vendor here.